MastodonSDK/Sources/MastodonUI/View/Content/MediaView.swift

```diff
@@ -118,18 +128,18 @@ extension MediaView {
         case .image(let info):
             layoutImage()
             bindImage(configuration: configuration, info: info)
-            accessibilityLabel = "Show image"       // TODO: i18n
+            accessibilityHint = "Expands the image. Double-tap and hold to show actions" // TODO: i18n
```

Comment:
You probably oversay them, can you please add keys in `Localizable.strings` and `app.json`? That would be awesome!